src/po/components/components.js

```diff
@@ -0,0 +1,40 @@
+export class TrelloComponents {
+  item(name) {
+    const selector = {
+      logIn: "//a[text()='Log in']",
+      userName: "#username",
+      continueBtn: "#login-submit",
+      password: "#password",
+      logInBtn: ".css-178ag6o",
+      account: ".B1uWdim9Jd0dJ9",
```

Comment:
where it's possible it's better to use test-data attributes 
[data-testid="header-member-menu-avatar"]
check please for other elements selector, where class looks like random text